phpmyfaq/src/phpMyFAQ/Permission/BasicPermission.php

```
@@ -275,4 +275,15 @@ public function refuseAllUserRights(int $userId): bool
     {
         return $this->repository->refuseAllUserRights($userId);
     }
+    /**
+     * Returns an array with the IDs of all groups the user belongs to.
+     * Since this is BasicPermission, always return an empty array.
+     *
+     * @param int $userId User ID
+     * @return array<int>
+     */
+    public function getUserGroups(int $userId): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$userId'. (undefined)

(UnusedFormalParameter)